src/com/team2502/robot2018/Constants.java

```diff
@@ -34,87 +32,78 @@
      */
 
     // Currently the max percent ft/s that can be given to each to each wheel
-    public static final float VR_MAX = 10F;
-    public static final float VL_MAX = 10F;
-    public static final float VR_MIN = -10F;
-    public static final float VL_MIN = -10F;
-
-    // The max change in voltage per second (acceleration)
-    public static final float AR_MAX = 5F;
-    public static final float AL_MAX = 5F;
-    public static final float AR_MIN = -10F;
-    public static final float AL_MIN = -10F;
 
-    public static InterpolationMap ACCELERATION_FOR_ELEVATOR_HEIGHT;
+    // DO NOTHING
+//    @Deprecated
+//    public static final float VR_MAX = 16F;
+//    @Deprecated
+//    public static final float VL_MAX = 16F;
+//    @Deprecated
+//    public static final float VR_MIN = -16F;
+//    @Deprecated
+//    public static final float VL_MIN = -16F;
 
+    // The max change in voltage per second (acceleration)
+//    public static final float AR_MAX = 4F;
+//    public static final float AL_MAX = 4F;
+//    public static final float AR_MIN = -4F;
+//    public static final float AL_MIN = -4F;
     // The lookahead distance (feet) for Pure Pursuit
-    public static final float LOOKAHEAD_MIN_DISTANCE_FT = 3F;
-    public static final float LOOKAHEAD_MAX_DISTANCE_FT = 5F;
+    public static final float LOOKAHEAD_MIN_DISTANCE_FT = 2F;
+    public static final float LOOKAHEAD_MAX_DISTANCE_FT = 5.5F;
     public static final float LOOKAHEAD_MIN_SPEED_FPS = 1F;
-    public static final float LOOKAHEAD_MAX_SPEED_FPS = 8F;
+    public static final float LOOKAHEAD_MAX_SPEED_FPS = 10F;
     public static final Lookahead LOOKAHEAD = new Lookahead(LOOKAHEAD_MIN_DISTANCE_FT, LOOKAHEAD_MAX_DISTANCE_FT,
                                                             LOOKAHEAD_MIN_SPEED_FPS, LOOKAHEAD_MAX_SPEED_FPS);
-    public static final float STOP_DIST_TOLERANCE_FT = 1F;
+    public static final float STOP_DIST_TOLERANCE_FT = 2F;
+    // TODO: figure out why wheel diameter has to be much smaller than it should be (normally 6)
```

Comment:
We have figured this out: it is a result of encoder positioning + gear reduction